client/web/src/search/input/SearchPage.tsx

```diff
@@ -130,207 +139,85 @@ export const SearchPage: React.FunctionComponent<SearchPageProps> = props => {
-                                        <dl className="col-xs-12 col-xl-6">
-                                            <dt className="search-page__help-content-subheading">
-                                                <h5>Finding matches</h5>
-                                            </dt>
-                                            <dd>
-                                                <p>
-                                                    <strong>Regexp:</strong>{' '}
-                                                    <span className="text-monospace">(read|write)File</span>
-                                                </p>
-                                            </dd>{' '}
-                                            <dd>
-                                                <p>
-                                                    <strong>Exact:</strong>{' '}
-                                                    <span className="text-monospace">"fs.open(f)"</span>
-                                                </p>
-                                            </dd>
-                                            <dd>
-                                                <p>
-                                                    <strong>Structural:</strong>{' '}
-                                                    <span className="text-monospace">if(:[my_match])</span>
-                                                </p>
-                                            </dd>
-                                        </dl>
+                            </div>
+
+                            <div className="mt-5 d-flex justify-content-center">
+                                <div className="d-flex align-items-center search-page__cta">
+                                    <SignUpCta />
+                                    <div className="mt-2">
+                                        Prefer a local installation?{' '}
+                                        <a
+                                            href="https://docs.sourcegraph.com"
+                                            target="_blank"
+                                            rel="noopener noreferrer"
+                                        >
+                                            Install Sourcegraph locally.
+                                        </a>
                                     </div>
                                 </div>
                             </div>
-                            <div className="row justify-content-center">
-                                <div className="mx-auto col-sm-12 col-md-8 col-lg-8 col-xl-6">
-                                    <PrivateCodeCta />
+
+                            <div className="mt-5">
+                                <div className="d-flex align-items-baseline mt-5 mb-3">
+                                    <h3 className="search-page__help-content-header mr-2">Repository pages</h3>
```

Comment:
This label doesn't sound right to me, these are "Repository groups", not single repos.

```suggestion
                                    <h3 className="search-page__help-content-header mr-2">Repository group pages</h3>
```